.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+version: 2.1
+jobs:
+  build:
+    machine:
+      image: circleci/classic:latest
+    steps:
+      - checkout
+  test:
+    working_directory: ~/program-service/src
+    docker:
+      - image: cimg/node:13.10.0
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: pw4ttpllt043
```

Comment:
@ahghatol No hardcoded secrets. Now this commit needs to be wiped from repo or we need to change the password to keep it a secret.